### DISCUSSION TITLE
inciar sección

### DISCUSSION THREAD

**Discussion by Marrantgangs** [community/community#136255] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Buenas noches supieran como puedo recuperar mi Facebook alguien entro a mi wifi y dispositivos y me robaron mis cuentas redes sociales y lo tengo desde hace 7 años 😔😔

**Comment by LiteBrite82** [comment]
Hi @Marrantgangs ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
